Dockerfile

```diff
@@ -0,0 +1,41 @@
+FROM phusion/baseimage
+MAINTAINER Erlang Solutions <mongoose-im@erlang-solutions.com>
+
+RUN useradd -ms /bin/bash amoc
+
+RUN apt-get update && \
+    apt-get install -y --no-install-recommends \
+        git \
+        make \
+        gcc \
+        clang \
+        libexpat1-dev \
+        wget \
+        iproute2 && \
+    wget https://packages.erlang-solutions.com/erlang-solutions_1.0_all.deb && \
+    dpkg -i erlang-solutions_1.0_all.deb && \
+    apt-get update && \
+    apt-get install -y esl-erlang=1:18.3.4
```

Comment:
When do we upgrade do 19.3?

Reply:
It is not scope of this PR. 
We should test it with 19.3 and create another PR.